tools/stanford/flowsheet_cleaner.py

```diff
@@ -215,6 +251,44 @@ def correct_rows(
                         "load_row_id": "",
                     }
                 )
+
+        with io.TextIOWrapper(
+            zstandard.ZstdCompressor(1).stream_writer(
+                open(destination_rel_path, "wb")
+            )
+        ) as o2:
+            rel_writer = csv.DictWriter(
+                o2,
+                fieldnames=[
+                    "concept_id_1",
```

Comment:
Why are we using `concept_id_1` and `concept_id_2` -- are these standard field names from BigQuery/OMOP? `child_concept_id` and `parent_concept_id` are much more descriptive.

Reply:
This is per the omop schema.